swan-lake/resources/featured-scenarios/handle-errors-and-replay-failed-activities-in-workflows.md

```
@@ -0,0 +1,197 @@
+
+The deposit succeeds with the corrected input, the customer is notified, and the workflow completes:
+
+```
+$ curl http://localhost:8080/payouts/<workflowId>
+{"workflowId":"...", "status":"COMPLETED", "result":"Claim CLM-001 paid. Deposit reference: DEP-ACC-12345"}
+```
+
+The samples repository also includes a minimal single-page React dashboard shared by all the workflow samples — <a href="https://github.com/ballerina-guides/integration-samples/tree/main/workflow-dashboard" target="_blank">`workflow-dashboard`</a>. Run it with this integration's task queue and open <a href="http://localhost:3000" target="_blank">http://localhost:3000</a>:
+
+```
+$ cd ../workflow-dashboard
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)